var/spack/repos/builtin/packages/libceed/package.py

```diff
@@ -31,8 +33,10 @@ class Libceed(Package):
     conflicts('+magma', when='@:0.5')
 
     depends_on('cuda', when='+cuda')
+    depends_on('hip@3.8.0', when='@0.7+hip')
 
     depends_on('occa@develop', when='@develop+occa')
+    depends_on('occa@1.1.0', when='@0.7:+occa')
```

Comment:
```suggestion
    depends_on('occa@1.1.0:', when='@0.7:+occa')
```

Reply:
We've had to track OCCA pretty closely due to interface changes so this seems to invite future breakage. We could do a narrower range, but I'm pretty sure we needed changes when tracking `1.0.*` subminor releases.